server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Mailbox.scala

```diff
@@ -54,11 +54,17 @@ object MailboxNamespace {
   def personal = PersonalNamespace
 }
 
-sealed trait MailboxNamespace
+sealed trait MailboxNamespace {
+  def asString: String
```

Comment:
do you expect that `asString` to be used outside json serialization?
If not, you should write it as a pattern-matching in Serializer.scala